flink-connectors/flink-connector-elasticsearch/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,57 @@
+This project includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+-------------------------------------------------------------
+
+This project bundles the following dependencies under
+the Apache Software License 2.0
+
+  - org.apache.lucene : lucene-core version 4.10.4
```

Comment:
isn't this list missing some dependencies? like jackson, joda-time, yaml, tartarus.

Reply:
You're right, there seems to be some missing from this list. Fixing ..